src/book/01-team/04-history/02-past-members.mdx

```diff
@@ -0,0 +1,389 @@
+
+_Insert team photo_
+
+#### Academic Supervisors and Mentors
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- **Dr. Alexandre Mendes**: Academic Supervisor of the Team
+- _Missing anyone?_
+
+#### Student Members
+
+- _Who went to RoboCup in 2016?_
+
+#### Supporting Members
+
+- _Missing anyone?_
+
+---
+
+## 2015: Hefei, China
+
+_Insert team photo_
+
+#### Academic Supervisors and Mentors
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- **Dr. Alexandre Mendes**: Academic Supervisor of the Team
+- _Missing anyone?_
+
+#### Student Members
+
+- _Who went to RoboCup in 2015?_
+
+#### Supporting Members
+
+- _Missing anyone?_
+
+---
+
+## 2014: Joao Pessoa, Brazil
+
+_Insert team photo_
+
+#### Academic Supervisors and Mentors
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- **Dr. Alexandre Mendes**: Academic Supervisor of the Team
+- _Missing anyone?_
+
+#### Student Members
+
+- _Who went to RoboCup in 2014?_
+
+#### Supporting Members
+
+- _Missing anyone?_
+
+---
+
+## 2013: Eindhoven, Netherlands
+
+_Insert team photo_
+
+#### Academic Supervisors and Mentors
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- **Dr. Alexandre Mendes**: Academic Supervisor of the Team
+- _Missing anyone?_
+
+#### Student Members
+
+- _Who went to RoboCup in 2013?_
+
+#### Supporting Members
+
+- _Missing anyone?_
+
+---
+
+## 2012: Mexico City, Mexico
+
+_Insert team photo_
+
+#### Academic Supervisors and Mentors
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- **Dr. Alexandre Mendes**: Academic Supervisor of the Team
+- _Missing anyone?_
+
+#### Student Members
+
+- _Who went to RoboCup in 2012?_
+
+#### Supporting Members
+
+- _Missing anyone?_
+
+---
+
+## 2011: Istanbul, Turkey
+
+_Insert team photo_
+
+#### Academic Supervisors and Mentors
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- **Dr. Alexandre Mendes**: Academic Supervisor of the Team
+- _Missing anyone?_
+
+#### Student Members
+
+- _Who went to RoboCup in 2011?_
+
+#### Supporting Members
+
+- _Missing anyone?_
+
+---
+
+## 2010: Singapore
+
+_Insert team photo_
+
+#### Academic Supervisors and Mentors
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- _Missing anyone?_
+
+#### Student Members
+
+- _Who went to RoboCup in 2010?_
+
+#### Supporting Members
+
+- _Missing anyone?_
+
+---
+
+## 2009: Graz, Austria
+
+_Insert team photo_
+
+#### Academic Supervisors and Mentors
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- _Missing anyone?_
+
+#### Student Members
+
+- _Who went to RoboCup in 2009?_
+
+#### Supporting Members
+
+- _Missing anyone?_
+
+---
+
+## 2008: Suzhou, China
+
+_Insert team photo_
+
+#### Academic Supervisors and Mentors
+
+- **Associate Professor Stephan Chalup**: Academic Head of the Team
+- _Missing anyone?_
+
+#### Student Members
+
+- _Who went to RoboCup in 2008?_
```

Comment:
Alexander Buckley, Stephan K. Chalup, Naomi Henderson,
Jason Kulk, Steven P. Nicklin, and ShekMan Tang,
Aaron S.W. Wong

Reply:
Naomi Henderson: Team Leader